var/www/html/consul-templates/db-config.php.ctmpl

```diff
@@ -0,0 +1,457 @@
+<?php
```

Comment:
Either way make sense to me. I felt like the webroot was a bit noisy with all of the templates there plus their rendered versions right next to the templates, can easily move them back to the webroot if that is what we want.